capstone/src/main/webapp/style.css

```diff
@@ -4,3 +4,50 @@
   height: 600px;
   width: 600px;
 }
+
+#main-content {
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
You can do the same in one line with -> margin: 0px auto

Reply:
Nit: no units on 0 :)
Yeah, I'll make the change.

Reply:
Ah you're right. Nice job.